gatewayconfig/logger.py

```diff
@@ -1,22 +1,21 @@
 import logging
 
-import os
+_log_format = f"%(asctime)s - [%(levelname)s] - %(name)s - (%(filename)s).%(funcName)s -- %(pathname)s:(%(lineno)d) - %(message)s"
 
-LOGLEVEL = os.environ.get("LOGLEVEL", "DEBUG")
-# create logger
-logger = logging.getLogger('gatewayconfig')
-logger.setLevel(LOGLEVEL)
 
-# create console handler and set level to debug
-stream_handler = logging.StreamHandler()
-stream_handler.setLevel(LOGLEVEL)
+def get_stream_handler():
+    stream_handler = logging.StreamHandler()
+    stream_handler.setLevel(logging.DEBUG)
```

Comment:
Lets continue to be able to overwrite with an environment variable `os.environ.get("LOGLEVEL", "DEBUG")`

Reply:
Done. https://github.com/NebraLtd/hm-config/blob/ca7b5e3e05041b9fdaaa238e3240c1f1e38f1cfd/gatewayconfig/logger.py#L10